### PR TITLE
utils/version.py: Streamline the suffix processing

### PR DESCRIPTION
**Pull request by mfalesni**
* Suffixes now have much more programmatical approach to parse
* Added some unittests for utils/version.py

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 9a873c02b426fb00ab139f92a711d20ff9ac654e:

`utils/tests/test_version.py`:
- :bangbang: Line 21:5: **E123** *closing bracket does not match indentation of opening bracket's line*

Please, rectify these issues :smirk: .
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 3d231bf185471b3392508878e875f61a3b96190b:

Everything seems all right :smile: .
*CFME QE Bot*

File: utils/tests/test_version.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+import pytest
+
+from utils.version import Version
+
+GT = '>'
```
Comment (RonnyPfannschmidt) [inline utils/tests/test_version.py]: how about using the operator module?

File: utils/version.py [hunk 2]
```diff
@@ -191,19 +193,49 @@ def parse(self, vstring):
...
+                suff_ver = 0
+            else:
+                suff_ver = int(suff_ver)
+            suff_t = self.SUFFIXES.index(suff_t)
+            numberized.append((suff_t, suff_ver))
+        return numberized
+
     @classmethod
     def latest(cls):
-        return cls('latest')
```
Comment (RonnyPfannschmidt) [inline utils/version.py]: cant we just assign those directly after class declaration?

File: utils/tests/test_version.py (hunk 1 above)
Comment (mfalesni) [inline utils/tests/test_version.py]: I wanted to have the operators visible in the test.

File: utils/version.py (hunk 2 above)
Comment (mfalesni) [inline utils/version.py]: Possibly, but that was not on the list yet.

**Pull request merged**